include/swift/Basic/Features.def

```diff
@@ -179,6 +179,7 @@ LANGUAGE_FEATURE(BuiltinAddressOfRawLayout, 0, "Builtin.addressOfRawLayout")
 LANGUAGE_FEATURE(MoveOnlyPartialConsumption, 429, "Partial consumption of noncopyable values")
 /// Enable bitwise-copyable feature.
 LANGUAGE_FEATURE(BitwiseCopyable, 426, "BitwiseCopyable protocol")
+SUPPRESSIBLE_LANGUAGE_FEATURE(ConformanceSuppression, 426, "Suppressible inferred conformances")
```

Comment:
Is ConformanceSuppression considered part of 426 or 428?

Reply:
It's part of 426: [06921cfe84d3a5025d296e0a462db8a9ad8ac413](https://github.com/apple/swift-evolution/blob/main/proposals/0426-bitwise-copyable.md#suppressing-inferred-conformance)